gradle.properties

```diff
@@ -8,17 +8,20 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
```

Comment:
These can be classified as `Pravega dependencies`, right?

Reply:
fixed